drivers/ma_yya_report/app/models/ma_yya_report/report.rb

```diff
@@ -119,7 +119,7 @@ def describe_filter_as_html
               )),
         A5d: a_t[:direct_assistance].eq(true).
           and(Arel.sql(
-                json_contains_text(:flex_funds, 'Rent arrears'),
+                json_contains_text(:flex_funds, 'Rent arrears'), # Change casing? Collected as 'Rent Arrears'
```

Comment:
Maybe better yet, we should update
```
private def json_contains_text(field, text)
   "#{field} ? '#{text}'"
end
```
to something that is case insensitive.  (I haven't found it yet though...)

Reply:
Oh. I didn't actually look at `json_contains_text` -- another bug is that it looks like currently "Rent arrears" would be scooped up by the "Rent" category, even though they are two separate categories of flex funds.